examples/full-jaffle-shop-demo/dbt/models/events.yml

```diff
@@ -86,6 +87,9 @@ models:
             in_gbp:
               type: sum
               format: 'gbp'
+            in_gbp_with_format_expression:
```

Comment:
Opened a ticket for the library to support big ints. https://github.com/borgar/numfmt/issues/61
I improved the logic in the util function so it handles big ints and strings. 
<img width="1064" alt="Screenshot 2025-02-11 at 11 34 38" src="https://github.com/user-attachments/assets/3b896f45-d531-4789-ace1-bc43c6d39b28" />